pkgs/dartpad_ui/web/index.html

```diff
@@ -80,26 +75,11 @@
       width: 100%;
     }
   </style>
-
-  <!-- This script adds the flutter initialization JS code -->
-  <script src="flutter.js" defer></script>
```

Comment:
or just `<script src="flutter_bootstrap.js" async></script>` here?

Dumping a bunch of JS in the file makes me...sad...